polly-merge.py

```diff
@@ -255,6 +281,46 @@ def get_comments():
         )
         return (pr_url, merge_ok)
 
+    def merge_after(match):
+        """Issue a merge if the matched url is merged"""
+        other_pr_url = match[1]
+        other_pr_url_stem = urllib.parse.urlsplit(other_pr_url)[2]
+
+        # basic sanity check URL is valid
+        # strip off anything post PR-ID (such as /overview /diff)
+        match = re.match(
+            "(/projects/.*/repos/.*/pull-requests/[0-9]*)[/.*]?", other_pr_url_stem
+        )
+        if not match:
+            return (pr_url, (False, f"invalid pr_url {other_pr_url}"))
+
+        if is_pr_merged(bitbucket_url, auth_header, match[1]):
+            return just_merge(match)
+        else:
+            return (pr_url, (False, f"{other_pr_url} not merged yet!"))
+
+    # dictionary of regex-pattern: command to run on match
+    commands = {
+        # command: merge
+        re.compile(f"{merge_trigger} merge$"): just_merge,
+        # command: merge-after <url>
+        re.compile(f"{merge_trigger} merge-after (.*)$"): merge_after,
+    }
+
+    def process_commands(list_to_check):
+        """Check for match and run command on list_to_check"""
+        for pattern, command in commands.items():
+            if match := list(filter(None, map(pattern.match, list_to_check))):
+                return command(match[0])
+        return None
+
+    # check PR description then comments. order is important; we don't want to
+    # do the relatively expensive fetch of comments for a PR if the description
+    # has a match
+    return process_commands([pr_data.get("description", "")]) or process_commands(
+        get_comments()
+    )
+
     return None
```

Comment:
Oops i left this dead code, good job [LGTM](https://lgtm.com/projects/g/noahp/polly-merge/snapshot/674e388e560a37bca53271fde4f3b09555401547/files/polly-merge.py?sort=name&dir=ASC&mode=heatmap#x1da4870546094d7e:1)
```suggestion
```